docs/grid_model/extensions.md

```diff
@@ -28,7 +30,7 @@ generator.newExtension(ActivePowerControlAdder.class)
     .add();
 ```
 
-The participation status and the participation factor are multi-variants: they can vary from one variant to another.
+The participation status, the participation factor, the maxP override and the minP override are multi-variants: they can vary from one variant to another.
```

Comment:
I think that maxP and minP are not in the variant, why do you need them to be in the variant?

Reply:
In general, in documentation we must be homogeneous: in the same table, we write Pmax and maxP.

Reply:
@annetill  this pMax/maxP story is indeed confusing.
I used the OLF convention (generator.setMaxP). The droop definition seems to use the ADN convention (pMax).
Let me know which one should be corrected (the droop definition or the use of the API convention fro the override)

Reply:
For variants - no strong argument for or against (use case may exist / but memory overhead) but we agreed to keep it in team meeting.
For doc, as said earlier, we agreed to keep the javaBean convention in the doc and use maxP / minP when talking about these quantities.